### DISCUSSION TITLE
[Feature request]: Add an option to hide snippets from subfolders

### DISCUSSION THREAD

**Discussion by LukasVolgger** [massCodeIO/massCode#295] [Ideas]
Currently when a subfolder is created in a folder, all snippets of the subfolder are displayed in the parent folder.

e.g.:

Folder 1 {snippet1}
.... subfolder 1 {snippet2}, {snippet3}
.... subfolder 2 {snippet4}, {snippet5}

Folder 1 {snippet1}, + (visible from subfolders) {snippet2}, {snippet3}, {snippet4}, {snippet5}

If I now have a very nested folder structure, it quickly becomes confusing here. An option to hide the snippets from subfolders would be a good solution.

**Comment by Alcance-PT** [comment]
The ability to hide the snippets from subfolders is a must to this product!

**Comment by KetschupMajo** [comment]
+1 - Would help a lot

**Comment by coma-toast** [comment]
+1 - I would like this feature as well.

**Comment by antonreshetov** [comment]
This feature has been implemented #371 and will be available in the next release
